bvcms/bvcms
# Josearmando pushpay authentication

Pushpay authentication, ready for code review.

Test flow:
1. Sign in as admin
2. Go to Administration/Setup/Pushpay
3. Sign in to your Pushpay admin account
4. Authorize
5. "Pushpay authorized" page displayed.
6. Go to Administration/Setup/Settings to confirm Access and Refresh token have been saved.

For this process to work a valid Touchpoint Auth Server Endpoing should be in place, I am using NGROK locally to have an Auth Test Server for this purpose, having my local has auth server endpoint should be taken as a test constraint. Open to discuss testing scenarios if necessary.

Webconfig appsetings for Pushpay must be set also.
